renovate.json

```diff
@@ -7,5 +7,9 @@
       "packageNames": ["node"],
       "enabled": false
     }
-  ]
+  ],
+  "prConcurrentLimit": 5,
+  "schedule": ["before 9am on monday"],
+  "stabilityDays": 3,
```

Comment:
This means that it waits 3 days after packages are released before creating update PR's. This should reduce updates from dependencies that release patch releases right after their main release